### ISSUE TITLE
Issue with CodeGlass on Chrome

### ISSUE THREAD

**Issue by kitsonk** [phiggins42/rstwiki#11]
I am having issues with rstwiki CodeGlass recent changes on Chrome 17.  I get the following in the console:

```
Resource interpreted as Script but transferred with MIME type text/x-js: "http://livedocs.dojotoolkit.org/_static/dojo/dojo.js".
```

And the CodeGlass pane come up blank and I don't get any other indication that there is an issue.  This is both my local installation and the livedocs one, but everything seems to be perfectly merry in FF10.

**Comment by wkeese** [comment]
Hmm, it's working for me (for example http://localhost:4200/dijit/Tree#a-programmatic-tree) at least on chrome 17/mac after my fix to your issue #12.   On the net tab I see that Content-Type  application/javascript

**Comment by kitsonk** [comment]
Hmmm... livedocs now seems to be working, maybe I was imagining it.  Ok, it seems to be something specific to me or my local rstwiki.
